lib/generators/komponent/templates/styleguide/components/index.js

```diff
@@ -1,3 +1,4 @@
+import "./style.css";
```

Comment:
I would prefer to name this file `_defaults.css`

Reply:
Sounds good to me.